api/envoy/api/v2/core/config_source.proto

```diff
@@ -96,6 +100,35 @@ message RateLimitSettings {
   google.protobuf.DoubleValue fill_rate = 2 [(validate.rules).double = {gt: 0.0}];
 }
 
+// Retry back off settings for discovery requests made by Envoy.
+message BackOffSettings {
+  // Jittered exponential back off algorithm.
+  message JitteredBackOff {
+    // The base interval to be used for the next back off computation. It should be greater than
+    // zero and less than or equal to :ref:`max_interval <max_interval>`.
+    google.protobuf.Duration base_interval = 1 [(validate.rules).message.required = true];
+
+    // The cap on the next back off value.
+    google.protobuf.Duration max_interval = 2 [(validate.rules).message.required = true];
+  }
+
+  // Fixed back off algorithm.
+  message FixedBackOff {
```

Comment:
Do we need fixed backoff or can we just be opinionated and say "jittered should be fine for every situation"?

Reply:
Square has some integration tests that could be sped up if we disabled exponential backoff, which is part of our motivation for working on this. I suppose we could use an exponential backoff with `base_interval = max_interval`, but when I wrote this I was using the `Envoy::BackOffStrategy` classes as a reference.

Reply:
Yeah, sounds better if we did `base_internal == max_interval` for your particular use case. One option is we make this `BackoffStrategy` and share it across the API, including in the route retry backoff. @envoyproxy/api-shepherds any other thoughts on this?

Reply:
That sounds fine to me.

Reply:
I'm not planning to change the route retry backoff in this PR, but if we want to reuse this message down the line, is there a better location for it?

Reply:
I'm tempted to say `envoy.types`. this used to be where just very basic types like `FractionalPercentage` lived, but I think this is growing. `envoy.api.v2.core` is fine as well, just in some separate proto. We'll be doing some package reorg in the v3 work, so if you just put it in some standalone proto called `backoff.proto`, we can find a home for it.